### ISSUE TITLE
level.dat file being constantly used and not being able to open or create worlds because of it

### ISSUE THREAD

**Issue by sinandds** [GTNewHorizons/GT-New-Horizons-Modpack#16001]
### Your GTNH Discord Username

xebecnyc

### Your Pack Version

2.5.1

### Your Server

SP

### Java Version

Java 21

### Type of Server

Single Player

### Your Actions

i was having issues with my pc bluescreening and it bluescreened while i was playing NH about a week ago ive since fixed the bluescreening, came back to nh and now my worlds wont load i cant create new worlds i cant load backups it also gives me a screen saying "level.dat is being used" 
![image (6)](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/55304298/978c99f5-28ca-4488-bbb4-c5c1787b892e)

### Crash Report

[fml-client-latest.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/15027573/fml-client-latest.log)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [ ] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by minehjon** [comment]
This is happening for me to, on java 8! if i restart the game (on prism launcher) and then spam yes for a bit, it lets me join the world! however, it resets my journeymap and baubles every time which is REALLY ANNOYING

**Comment by chochem** [comment]
use a backup from before. if that doesnt work then reinstall+use a backup from before.

**Comment by minehjon** [comment]
from before what? I never bluescreened like OP did. I just loaded a backup and it wouldnt let me join at all (on technic launcher), and when i tried the un-backup'ed version, neither did that or even a new world. I got a tip from the discord to move it over to a fresh install on prism launcher, so i did that and that made spam clicking yes work, where it didn't on technic.

I had tried reinstalling on technic before i moved to prism
